### PR TITLE
Create a zenodo metadata file

### PR DESCRIPTION
**Pull request by eric-wieser**
Based off the one from pygae/clifford.

I just added the zenodo webhook, so it should end up published after we do our next release.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    {
+      "name": "{The Pygae Team}"
+    }
+  ],
+  "keywords": [
+    "python",
+    "clifford algebra",
+    "geometric algebra",
+    "computer algebra",
+    "geometry"
```
Comment (utensil) [inline .zenodo.json:31]: Maybe add `symbolic`?

Comment (eric-wieser) [reply .zenodo.json:31]:
```suggestion
    "symbolic algebra",
    "geometry"
```
or
```suggestion
    "symbolic",
    "geometry"
```
or did you have something else in mind?

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "license": {
+    "id": "BSD-3-Clause"
+  }, 
+  "creators": [
+    {
+      "name": "Alan Bromborsky"
+    },
+    {
+      "name": "Utensil Song"
```
Comment (utensil) [inline .zenodo.json:10]: Just register an orcid: 0000-0003-3994-4466

Comment (eric-wieser) [reply .zenodo.json:10]:
```suggestion
      "name": "Utensil Song",
      "orcid": "0000-0003-3994-4466"
```

File: .zenodo.json (hunk 1 above)
Comment (utensil) [reply .zenodo.json:31]: symbolic on it own would be enough. mixed with algebra will interfere with the search I guess

Comment (eric-wieser) [reply .zenodo.json:31]: I'll see which zenodo has more matches for

Comment (utensil) [reply .zenodo.json:31]: image

It seems there's nothing more left. "Symbolic" is one particular feature of GAlgebra that is different from Clifford.

Comment (eric-wieser) [reply .zenodo.json:31]: https://zenodo.org/search?page=1&size=20&q=keywords:%22symbolic%22 has useless results, https://zenodo.org/search?page=1&size=20&q=keywords:%22symbolic%20algebra%22 looks at least vaguely sensible.

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "name": "{The Pygae Team}"
+    }
+  ],
+  "keywords": [
+    "python",
+    "clifford algebra",
+    "geometric algebra",
+    "computer algebra",
+    "symbolic algebra",
```
Comment (eric-wieser) [inline .zenodo.json:32]:
```suggestion
    "symbolic computation",
```

Comment (eric-wieser) [reply .zenodo.json:32]: has better matches: https://zenodo.org/search?page=1&size=20&q=keywords:%22symbolic%20computation%22

File: .zenodo.json (hunk 1 above)
Comment (utensil) [reply .zenodo.json:34]: I see that even SymPy didn't label itself with this...

Comment (utensil) [reply .zenodo.json:34]: symbolic expression gives me some reasonable results

File: .zenodo.json (hunk 3 above)
Comment (utensil) [reply .zenodo.json:32]: +1 for symbolic computation

File: .zenodo.json (hunk 1 above)
Comment (eric-wieser) [reply .zenodo.json:34]: Will add that later. Gave up yesterday because zenodo wouldn't load for me any more.

**Pull request merged**